model/gurps/skill.go

```diff
@@ -732,8 +733,10 @@ func (s *Skill) DecrementSkillLevel() {
 func (s *Skill) CalculateLevel(excludes map[string]bool) Level {
 	points := s.AdjustedPoints(nil)
 	if s.IsTechnique() {
-		return CalculateTechniqueLevel(EntityFromNode(s), s.Replacements, s.NameWithReplacements(),
-			s.SpecializationWithReplacements(), s.Tags, s.TechniqueDefault, s.Difficulty.Difficulty, points, true,
+		e := EntityFromNode(s)
+		requirePoints := !e.SheetSettings.UseSkillTrees
```

Comment:
Again, the use of this seems to be inverting the current logic, which is incorrect.